typings/Picker.d.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from "react"
+import { TextStyle, StyleProp, ViewProps } from 'react-native'
+
+type ItemValue  = number | string
+
+export interface PickerItemProps {
+	label?: string;
+	value: ItemValue;
+	color?: string;
+	testID?: string;
+}
+
+class PickerItem extends React.Component<PickerItemProps, {}> {}
```

Comment:
Reflecting changes from DefinitelyTyped/DefinitelyTyped#41502

This line causes TypeScript users to be able to import PickerItem which isn't the component's name and may lead to confusion.

```suggestion
```

Reply:
done